models/labels/addresses/__single_category_labels__/cex_users/labels_cex_users.sql

```diff
@@ -10,44 +10,32 @@
     'ethereum'
 ] %}
 
-SELECT 
-blockchain, address, name, category, contributor, source, created_at, updated_at, model_name, label_type
+SELECT blockchain,
+       address,
+       name,
+       category,
+       contributor,
+       source,
+       created_at,
+       updated_at,
+       model_name,
+       label_type
 FROM (
 {% for chain in chains %}
     SELECT
-    '{{chain}}' as blockchain,
-    address,
-    cex_name || ' User' AS name,
-    'cex users' AS category,
-    'msilb7' AS contributor,
-    'query' AS source,
-    timestamp('2023-03-11') as created_at,
-    now() as updated_at,
-    'cex_users_withdrawals' model_name,
-    'persona' as label_type
-
-    FROM {{source('erc20_' + chain, 'evt_transfer')}} t
-        INNER JOIN {{ref('addresses_'+ chain +'_cex')}} c
-        ON t.`from` = c.address
-
-    UNION ALL
-
-    SELECT
-    '{{chain}}' as blockchain,
-    address,
-    cex_name || ' User' AS name,
-    'cex users' AS category,
-    'msilb7' AS contributor,
-    'query' AS source,
-    timestamp('2023-03-11') as created_at,
-    now() as updated_at,
-    'cex_users_withdrawals' model_name,
-    'persona' as label_type
-
-
+        '{{chain}}' as blockchain,
+        address,
+        cex_name || ' User' AS name,
+        'cex users' AS category,
+        'msilb7' AS contributor,
+        'query' AS source,
+        timestamp('2023-03-11') as created_at,
+        now() as updated_at,
+        'cex_users_withdrawals' model_name,
+        'persona' as label_type
     FROM {{source('erc20_' + chain, 'evt_transfer')}} t
```

Comment:
The pre-commit check fails because of this jinja line, but the compiled target spell works fine.

Reply:
It would be cool to allow table name string concatenation in tests. if only the tests are failing but the models work fine, is it fine to get it merged @jeff-dude? Else I can wait for #3411 to be merged and use the spells created there instead of concatenation

Also, this spell here was already here and working fine, it was created by Michael before this PR

Reply:
yeah, we've been bypassing this issue, that's fine. we'll have to extend the customization of the pre-commit checks to handle this scenario in the future